test/govspeak_test.rb

```diff
@@ -1247,6 +1247,20 @@ class GovspeakTest < Minitest::Test
     )
     end
 
+  test_given_govspeak "
```

Comment:
**thought:** documentation

Would be nice if this method took some kind of human-readable description argument, but now's probably not the time to go adding that to this massive spec!

Reply:
It would be nice to break this huge file down into some smaller ones, or even just wrap some context blocks around things.  I'll add a card to the backlog so it's not forgotten, as I don't think it'll be a trivial task to understand what exactly is being tested in each example.